.github/workflows/spm.yml

```diff
@@ -51,7 +51,7 @@ jobs:
     strategy:
       matrix:
         target: [tvOS, macOS, catalyst]
-        # Full set of Firebase-Package tests only run on iOS because of Analytics.
+        # Include non iOS tests in this section.
```

Comment:
Still an issue with FIAM and macOS: `FIRInAppMessageDisplayStoryboard.storyboard: error: iOS storyboards do not support target device type "mac".`

How about leave line 54 and only delete `because of Analytics` ?